rules/experimental/rbe/exec_properties.bzl

```diff
@@ -13,6 +13,96 @@
 # limitations under the License.
 
 """This file contains macros to create and manipulate dictionaries of properties to be used as execution properties for RBE.
+
+It also contains macros that create repository rules for standard and custom sets of execution
```

Comment:
Should we have this statement first and the "macros to create and manipulate dictionaries of properties" second?

Reply:
done